docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -0,0 +1,116 @@
+  [managed credentials integration](/docs/products/clickhouse/concepts/data-integration-overview#managed-credentials-integration)
+
+## Observability integrations vs data source integrations
+
+Aiven for ClickHouse supports observability integrations and data source integrations,
+which have different purposes:
+
+- [Observability integration](/docs/products/clickhouse/howto/list-integrations) is
+  connecting to other services (either in-Aiven or external) to expose and process logs
+  and metrics.
+- Data service integration is connecting to other services (either in-Aiven or external)
+  to use them as data sources. In Aiven for ClickHouse, data service integration is
+  possible with the following data source types:
+
+  - Apache Kafka®
+  - PostgreSQL®
+  - MySQL®
+  - ClickHouse®
+  - Amazon S3®
+
+## In-Aiven integrations vs external integrations
+
+By enabling data service integrations, you create streaming data pipelines across
+services. Depending on where the services are located, you can have either in-Aiven
+integrations (between Aiven services) or external integrations (between an Aiven service
+and an external data source or application).
+
+For integrating with external data sources, Aiven for ClickHouse provides two types of
+data service integrations:
+
+- [Managed databases](/docs/products/clickhouse/concepts/data-integration-overview#managed-databases-integration)
+- [Managed credentials](/docs/products/clickhouse/concepts/data-integration-overview#managed-credentials-integration)
+
+## Managed credentials integration
+
+The managed credentials integration uses the
+[ClickHouse named collections](https://clickhouse.com/docs/en/operations/named-collections)
+logic. It allows storing key-value pairs that are to be used as credentials when
+connecting to external data sources. To integrate data, you create tables using table
+engines. With the managed credentials integration enabled,
+querying the data is easier and quicker since you no longer need connections parameters in
+each query. They are stored and available from in-Aiven credential storage.
+
+Managed credentials integration in Aiven for ClickHouse is supported with the following
+data source types:
+
+- PostgreSQL®
+- MySQL®
+- Amazon S3®
+
+:::important
+The managed credentials integration works one-way: It allows to integrate with data source
+and access your data from Aiven for ClickHouse, but it doesn’t allow to access Aiven for
+ClickHouse data from the integrated data source.
+:::
+
+:::note[See also]
+For information on how table engines work in Aiven for ClickHouse services, preview
+[Engines: database and table](/docs/products/clickhouse/concepts/service-architecture#engines-database-and-table).
+For the list of table engines available in Aiven for ClickHouse, check
+[Supported table engines](/docs/products/clickhouse/reference/supported-table-engines).
+:::
```

Comment:
We've removed all variations of "See also" admonitions to limit visual clutter and limit the options available to us to share links.  Could we use a straight up paragraph to keep the important note as the only visually important element? (applies to the other See also admonition below).